modules/acm_sku_stock/acm_sku_stock.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Acquia Commerce SKU Stock
+type: module
+description: This module adds stock management functionality to the Acquia Commerce Connector.
+core: 8.x
+package: Commerce
+dependencies:
+  - acm_commerce:acm_sku
```

Comment:
this should be `acquia_commercemanager:acm_sku`

Reply:
removing this as discussed in another comment